### PR TITLE
Refactor & 2018 update Amethyst Network

### PR DESCRIPTION
**Pull request by TimonPost**

### CODE REVIEW

File: amethyst_network/src/network_socket.rs [hunk 1]
```diff
@@ -9,12 +9,11 @@ use std::{
 };
 
 use amethyst_core::specs::{Join, Resources, System, SystemData, WriteStorage};
-use laminar::error;
-use laminar::net::UdpSocket;
-use laminar::{DeliveryMethod, NetworkConfig, Packet};
+use laminar::{ DeliveryMethod, NetworkConfig, Packet, net::UdpSocket};
```
Comment (khionu) [inline amethyst_network/src/network_socket.rs]: The space in front of `DeliveryMethod` is inconsistent with the rest

**Comment by TimonPost** [comment]
Can this be approved and merged?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+resources:
```
Comment (LucioFranco) [inline azure-pipelines.yml]: should this file live in this repo?

Comment (TimonPost) [reply azure-pipelines.yml]: Nope, it should not let me fix that!

**Comment by khionu** [comment]
@LucioFranco can you approve and merge?

**Comment by TimonPost** [comment]
bors r=udoprog,khionu, LucioFranco

**Pull request merged**